atd-toolbox/coordination_partners_migration/secrets.py

```diff
@@ -0,0 +1,10 @@
+HASURA_AUTH = {
+  "hasura_graphql_endpoint": {
+    "staging": "<redacted>",
+    "prod": "<redacted>",
```

Comment:
Thank you for redacting these, and I am sorry for being so late to the party in time to leave you a helpful review. In the future, if you have a file that contains secrets, I'd consider the pattern of putting that file into your `.gitignore` file so it just can't get put into the repo. Then, add a file like `secrets.py_template` with the shape of the file but the secrets blanked out so people know what the file they need looks like but have to provide their own secrets.

With this pattern, it's too easy for one of us to unredact it and then push it up to the repo.

Reply:
thats helpful feedback Frank -- we have this pattern of a secrets file in moped-toolbox and I get paranoid that im accidentally going to push up the secrets to the repo when I'm testing against staging or prod

Reply:
thats a great tip Frank and it makes a lot of sense. i will definitely take up that pattern in the future!